src/main/java/frc/robot/commands/LevelRobot.java

```diff
@@ -0,0 +1,61 @@
+package frc.robot.commands;
+
+import com.ctre.phoenix.sensors.WPI_Pigeon2;
+
+import edu.wpi.first.math.MathUtil;
+import edu.wpi.first.math.controller.PIDController;
+import edu.wpi.first.math.geometry.Translation2d;
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.Constants;
+import frc.robot.subsystems.ExampleSubsystem;
+import frc.robot.subsystems.Swerve.DrivebaseSubsystem;
+public class LevelRobot extends CommandBase{
+    private final DrivebaseSubsystem drive;
+    private final PIDController pid = new PIDController(
+        Constants.LevelRobotConstants.PID_kP, 
+        Constants.LevelRobotConstants.PID_kI, 
+        Constants.LevelRobotConstants.PID_kD);
+    private final PIDController headingPid = new PIDController(
+        Constants.LevelRobotConstants.HEADING_PID_kP,
+        Constants.LevelRobotConstants.HEADING_PID_kI,
+        Constants.LevelRobotConstants.HEADING_PID_kD);
+    private double startingYaw;
+    
+    public LevelRobot(SwerveSubsystem drivetrain){
+        drive = drivetrain;
+        addRequirements(drive);
+    }
+    // Called when the command is initially scheduled.
+    @Override
+    public void initialize() {
+        pid.setTolerance(0);
```

Comment:
This most likely will need to be changed on the full robot. The material is also different from what we have been testing on, so that will also change constants.

Reply:
Will so!